lib/main_ltp.pm

```diff
@@ -32,10 +33,11 @@ sub load_kernel_tests {
     loadtest_kernel "../installation/bootloader" if is_pvm;
 
     if (get_var('INSTALL_LTP')) {
-        if (is_alp) {
-            loadtest 'microos/disk_boot';
+        if (is_transactional) {
+            is_s390x ? loadtest 'boot/boot_to_desktop' : loadtest 'microos/disk_boot';
```

Comment:
Could you include why this is different in commit message? I guess it is because s390x is not on KVM. Also does this effect other backends?